docs/notebooks/01_merge.ipynb

```diff
@@ -1,27 +1,505 @@
 {
```

Comment:
<p>I am not sure how to best handle the age column for validation, maybe you have a better idea?!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/icbi-lab/atlas_protocol/pull/19/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>maybe define a <code>min</code> and <code>max</code> value in yaml...  </p>